### ISSUE TITLE
Scalding Mornbrew Delivery autocompletes

### ISSUE THREAD

**Issue by vmensik** [chromiecraft/chromiecraft#460]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [x] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

The dwarf quest Scalding Mornbrew Delivery (https://wowgaming.altervista.org/aowow/?quest=3364) is instantly marked as completed after pickup and no timer is present. 

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The quest should be timed (5 min) and complete after you reach the required position.
https://classic.wowhead.com/quest=3364/scalding-mornbrew-delivery#videos:id=166317

##### STEPS TO REPRODUCE THE PROBLEM:

1. Pick up quest
2. Look at quest status

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/b8b5f68f98ff8771a6ec87ea6805ceced91c406a

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Azcobu** [comment]
Did some more checking and found a comment at https://www.wowhead.com/quest=3364/scalding-mornbrew-delivery#comments:id=926999 that says this quest was no longer timed as of patch 3.3. If this is accurate, and the other comments don't dispute it, then this quest is actually behaving correctly and the wording is just a bit ambiguous.

I'll leave this open for a day or so in case anyone else has comments and then close the issue as Working As Intended.

**Comment by Azcobu** [comment]
Closing as per above.

**Comment by cachocarne** [comment]
CONFIRMED.
I was a bit surprised after doing it today and noticing there was no timer. 
[Classic Wowhead](https://classic.wowhead.com/quest=3364/scalding-mornbrew-delivery) says it was. [Wowpedia](https://wowpedia.fandom.com/wiki/Scalding_Mornbrew_Delivery_(quest)) says it was.

However, I also found the [comment on Wowhead](https://www.wowhead.com/quest=3364/scalding-mornbrew-delivery) stating that the 5 min timer was removed on 3.3, quite lame. You know "its part of blizz's plan to make WoW nicer for newbies, all the mobs in starting areas are yellow, timers on quests gone, warlock imp quests gone and quests automatically accepted". We just couldn't see back then where all this was heading.
